app/src/main/java/com/andrew/tasky/presentation/views/TimeDateSelectorCustomView.kt

```diff
@@ -0,0 +1,17 @@
+package com.andrew.tasky.presentation.views
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.constraintlayout.widget.ConstraintLayout
+import com.andrew.tasky.R
+import com.andrew.tasky.databinding.CvTimeDateSelectorBinding
+
+class TimeDateSelectorCustomView@JvmOverloads constructor(
+        context: Context,
+        attrs: AttributeSet? = null
+    ): ConstraintLayout(context, attrs) {
+
+        private val binding = CvTimeDateSelectorBinding.bind(
+            inflate(context, R.layout.cv_time_date_selector, this)
+        )
+    }
```

Comment:
Is this your whole custom view or will you extend this with some functionality?

Reply:
I think this is the only functionality I was gonna use for this

Reply:
Then you don't need a custom view for this, you can instead use this in your XML layout to get a reference to the corresponding binding:
```
<include android:id="@+id/timeDateSelectorView" layout="@layout/your_layout_here" />
```